differential_drive_controller:
  ros__parameters:
    wheel_base: 0.6           # Distance between wheels (meters)
    wheel_radius: 0.15        # Radius of the wheels (meters)
    max_linear_speed: 1.2            # Max allowable wheel speed (m/s)
    max_angular_speed: 1.0           # Max allowable angular speed (rad/s)
    linear_kp: 1.0            # PID parameters for linear velocity
    linear_ki: 0.0
    linear_kd: 0.0
    angular_kp: 1.0           # PID parameters for angular velocity
    angular_ki: 0.0
    angular_kd: 0.0
    controller_frequency_: 100.0          # Rate at which to publish the control commands (Hz)
